src/main/app/src/pages/koulutus/KoulutusForm/JarjestajaSection.tsx

```diff
@@ -45,9 +49,12 @@ const OrganizationSection = ({
   );
 
   const getIsDisabled = useCallback(
-    organisaatio =>
-      !roleBuilder.hasUpdate(KOULUTUS_ROLE, organisaatio).result(),
-    [roleBuilder]
+    organisaatio => {
+      const kt = koulutus ? koulutus.koulutustyyppi : 'unknown';
+      const requiredRole = kt === 'amm' ? OPH_PAAKAYTTAJA_ROLE : KOULUTUS_ROLE;
```

Comment:
Kannattaa käyttää `KOULUTUSTYYPPI`-enumin arvoa (constants.ts-tiedostosta) merkkijonon sijaan.
```suggestion
      const requiredRole = kt === KOULUTUSTYYPPI.AMMATILLINEN_KOULUTUS ? OPH_PAAKAYTTAJA_ROLE : KOULUTUS_ROLE;
```